devops/chocolatey-test-environment.nuspec

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>chocolatey-test-environment</id>
+    <version>0.10.15</version>
+    <packageSourceUrl>https://github.com/Heteroculturalism/chocolatey-packages</packageSourceUrl>
+    <owners>Jerome Bell</owners>
```

Comment:
I think this might need adjusted to also have Chocolatey contributors.

Reply:
What exact contributors should I add?

Reply:
Bump